scripts/pytest-image.Dockerfile

```diff
@@ -13,11 +13,15 @@ ARG TEST_INFRA_PATH=./test-infra
 # Destination path to copy aws-web-identity-token
```

Comment:
nit: the other Dockerfiles in [`prow/jobs/images`](https://github.com/aws-controllers-k8s/test-infra/tree/main/prow/jobs/images) all use the form `Dockerfile.$suffix` instead of `$prefix.Dockerfile`